ppx/test/pp_ocaml.expected

```diff
@@ -16,11 +16,8 @@ let make =
               [|("key",
                   (inject
                      (Js_of_ocaml.Js.Optdef.option
-                        (Option.map Js_of_ocaml.Js.string key))));("name",
```

Comment:
Did anything change in the tests? Hard to tell with the refmt changes.

Reply:
Just the Module added